setup.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from typing import List, Union
+
+from setuptools import find_packages, setup
+
+
+def load_requirements(file: Union[str, Path]) -> List[str]:
+    path = file if isinstance(file, Path) else Path(file)
+    requirements: List[str] = []
+    if not path.is_file():
+        raise FileNotFoundError(f"path does not point to a valid file: {path}")
+    if not path.suffix == ".in":
+        raise ValueError(f"file should have .in extension: {path}")
+    file_dir = path.parent.resolve()
+    with open(file, encoding="utf-8") as f:
+        raw = f.read().splitlines()
+        requirements.extend([r for r in raw if not r.startswith("#") and not r.startswith("-")])
+        recursive_reqs = [r for r in raw if r.startswith("-r")]
+    for recursive_req in recursive_reqs:
+        file_spec = recursive_req.split()[-1]
+        file_path = Path(file_dir) / file_spec
+        requirements.extend(load_requirements(file=file_path.resolve()))
+    # Remove duplicates and any blank entries
+    return list({r for r in requirements if r})
+
+
+setup(
+    name="unstructured-platform-plugins",
+    version="1.0.0",
+    python_requires=">=3.12,<3.13",
```

Comment:
This limits it to 3.12, which is quite a new python version. Does the code contain elements which are incompatible with 3.10 or 3.11?

Reply:
If there's a reason to support prior to `3.12`, we could expand this range.